Samples/Mutation_Files/crossover.py

```diff
@@ -0,0 +1,37 @@
+import random
+
+def single_point_crossover(timetable1, timetable2):
+    # Ensure both parents have the same length
+    assert len(timetable1) == len(timetable2), "Parents must have the same length"
+    
+    # Choose a random crossover point
+    crossover_point = random.randint(1, len(timetable1) -1)
+    
+    # Generate offspring by swapping genes at the crossover point
```

Comment:
Make sure to increase the genetic diversity

Reply:
that will depend on the  diversity of the data given